<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-02-13T00:03:18 IST" name="Default suite" finished-at="2025-02-13T00:03:46 IST" duration-ms="28248">
    <groups>
    </groups>
    <test started-at="2025-02-13T00:03:18 IST" name="Default test" finished-at="2025-02-13T00:03:46 IST" duration-ms="28248">
      <class name="library.Assignment5">
        <test-method signature="enableTrueCase()[pri:0, instance:library.Assignment5@69ee81fc]" started-at="2025-02-13T00:03:18 IST" name="enableTrueCase" finished-at="2025-02-13T00:03:28 IST" duration-ms="10661" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enableTrueCase -->
        <test-method signature="timeoutCase()[pri:0, instance:library.Assignment5@69ee81fc]" started-at="2025-02-13T00:03:28 IST" name="timeoutCase" finished-at="2025-02-13T00:03:33 IST" duration-ms="5024" status="FAIL">
          <exception class="org.testng.internal.thread.ThreadTimeoutException">
            <message>
              <![CDATA[Method library.Assignment5.timeoutCase() didn't finish within the time-out 5000]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method library.Assignment5.timeoutCase() didn't finish within the time-out 5000
at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:211)
at java.base@17.0.7/java.util.concurrent.CompletableFuture$Signaller.block(CompletableFuture.java:1864)
at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
at java.base@17.0.7/java.util.concurrent.CompletableFuture.waitingGet(CompletableFuture.java:1898)
at java.base@17.0.7/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2072)
at platform/java.net.http@17.0.7/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:553)
at platform/java.net.http@17.0.7/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
at app//org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute(JdkHttpClient.java:292)
at app//org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:181)
at app//org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at app//org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at app//org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
at app//org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:297)
at app//library.Assignment5.timeoutCase(Assignment5.java:77)
at java.base@17.0.7/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@17.0.7/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base@17.0.7/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base@17.0.7/java.lang.reflect.Method.invoke(Method.java:568)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base@17.0.7/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.thread.ThreadTimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- timeoutCase -->
        <test-method signature="priority1Case()[pri:1, instance:library.Assignment5@69ee81fc]" started-at="2025-02-13T00:03:33 IST" name="priority1Case" finished-at="2025-02-13T00:03:46 IST" duration-ms="12447" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- priority1Case -->
      </class> <!-- library.Assignment5 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
